src/ZombieDriver.php

```diff
@@ -94,13 +94,14 @@ public function stop()
     public function reset()
     {
         $js = <<<JS
-browser.destroy();
-browser = null;
```

Comment:
It would really cool if JavaScript opened popup windows won't affect what currently active window is from driver perspective and even though for user the popup seems to be blocking the main window all executed Mink code still goes to main window. Unfortunately there is no test for that either.